tools/rsdl/dotnet/rapid/sample.rsdl

```diff
@@ -1,3 +1,5 @@
+namespace "jetsons"
```

Comment:
I'd prefer no quotes, namespaces follow the identifier rules, they are not string values

Reply:
Understood. Will fix today.